src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2188,44 +2188,6 @@ internal static SpecialType GetEnumPromotedType(SpecialType underlyingType)
         }
 #nullable disable
 
-        private static BinaryOperatorKind SyntaxKindToBinaryOperatorKind(SyntaxKind kind)
-        {
-            switch (kind)
```

Comment:
moved to OperatorFacts to help out.

Reply:
> moved to OperatorFacts to help out.

Please leave the helper here in it original form, just make it accessible to other components